tests/Geocoder/Tests/Provider/GeocoderCaProviderTest.php

```diff
@@ -203,10 +203,11 @@ public function testGetReversedDataWithRealCoordinates()
         $this->assertCount(1, $result);
 
         $result = $result[0];
+        error_log(var_export($result, true));
         $this->assertInternalType('array', $result);
         $this->assertEquals(40.707507, $result['latitude'], '', 0.0001);
         $this->assertEquals(-74.011255, $result['longitude'], '', 0.0001);
-        $this->assertEquals('2', $result['streetNumber']);
+        $this->assertEquals('1', $result['streetNumber']);
```

Comment:
No - but due to a back end data change it was stopping the unit tests from running clean so I tweaked the expected results. I always try to isolate changes to a specific branch but as this was a single line change I broke my own rules in this case.